sdk/test-utils/recorder/package.json

```diff
@@ -65,40 +58,72 @@
   "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/test-utils/recorder/",
   "sideEffects": false,
   "dependencies": {
+    "@azure/core-client": "^1.0.0",
     "@azure/core-rest-pipeline": "^1.13.0",
     "@azure/core-auth": "^1.3.2",
     "@azure/core-util": "^1.6.1",
     "@azure/logger": "^1.0.0"
   },
   "devDependencies": {
-    "@azure/core-client": "^1.0.0",
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@types/express": "^4.16.0",
     "@types/fs-extra": "^8.0.0",
     "@types/chai": "^4.1.6",
-    "@types/mocha": "^10.0.0",
     "@types/node": "^18.0.0",
     "c8": "^8.0.0",
     "chai": "^4.2.0",
     "concurrently": "^8.2.0",
     "cross-env": "7.0.3",
-    "dotenv": "^16.0.0",
-    "eslint": "^8.0.0",
     "express": "^4.16.3",
-    "karma": "^6.2.0",
-    "karma-chrome-launcher": "^3.0.0",
-    "karma-coverage": "^2.0.0",
-    "karma-env-preprocessor": "^0.1.1",
-    "karma-firefox-launcher": "^1.1.0",
-    "karma-junit-reporter": "^2.0.1",
-    "karma-mocha": "^2.0.1",
-    "karma-mocha-reporter": "^2.2.5",
-    "karma-sourcemap-loader": "^0.3.8",
-    "mocha": "^10.0.0",
     "rimraf": "^5.0.5",
-    "tslib": "^2.2.0",
-    "ts-node": "^10.0.0",
-    "typescript": "~5.3.3"
-  }
+    "tslib": "^2.6.2",
+    "@vitest/browser": "^1.3.1",
+    "eslint": "^8.56.0",
+    "playwright": "^1.41.2",
+    "prettier": "^3.2.5",
+    "tshy": "^1.11.1",
+    "typescript": "~5.3.3",
+    "vitest": "^1.3.1",
+    "loupe": "^3.1.0",
+    "@azure-tools/vite-plugin-browser-test-map": "^1.0.0",
```

Comment:
Can we alphabetize the list here to make it a little bit more readable?

Reply:
updated